Packs/CoreAlertFields/.pack-ignore

```diff
@@ -11,4 +11,7 @@ ignore=IF100
 ignore=IF115
 
 [file:incidentfield-User_Risk_level.json]
-ignore=IF115
\ No newline at end of file
+ignore=IF115
+
+[file:incidentfield-Alert_Search_Results.json]
+ignore=IF100
```

Comment:
What 'IF100' representing?

Reply:
The word 'Alert' cannot be included in the incident field name.